client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,48 @@
+import { User } from '../../interface/User';
+import useStyles from './useStyles';
+import { Button, Card, Typography, Avatar, CardActions, Paper } from '@material-ui/core';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const ProfilePhoto = ({ loggedInUser }: Props): JSX.Element => {
```

Comment:
You can access the loggedInUser from context as opposed to passing down as props

Reply:
accessed loggedInUser directly in the component